### PR TITLE
Es 5.6.0.0

### PR DESCRIPTION
**Pull request by vvanholl**

### CODE REVIEW

**Comment by vvanholl** [comment]
FYI I do not longer use Jenkins CI

**Comment by lukas-vlcek** [comment]
Cool, I will look at this tomorrow in the morning.

File: src/main/java/org/compuscene/metrics/prometheus/PrometheusMetricsCollector.java [hunk 1]
```diff
@@ -629,6 +632,11 @@ private void registerIngestMetrics() {
         catalog.registerNodeGauge("ingest_pipeline_total_time_seconds", "Ingestion total time in seconds", "pipeline");
         catalog.registerNodeGauge("ingest_pipeline_total_current", "Ingestion total current", "pipeline");
         catalog.registerNodeGauge("ingest_pipeline_total_failed_count", "Ingestion total failed", "pipeline");
+
+        catalog.registerNodeGauge("ingest_pipeline_processor_total_count", "Ingestion total number", "pipeline", "processor");
+        catalog.registerNodeGauge("ingest_pipeline_processor_total_time_seconds", "Ingestion total time in seconds", "pipeline", "processor");
+        catalog.registerNodeGauge("ingest_pipeline_processor_total_current", "Ingestion total current", "pipeline", "processor");
+        catalog.registerNodeGauge("ingest_pipeline_processor_total_failed_count", "Ingestion total failed", "pipeline", "processor");
```
[the remark this answers is not in the document]
Comment (vvanholl) [reply src/main/java/org/compuscene/metrics/prometheus/PrometheusMetricsCollector.java]: It's done, can you make a final review before merging ?

[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply src/main/java/org/compuscene/metrics/prometheus/PrometheusMetricsCollector.java]: LGTM

**Pull request merged**